docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,265 @@
+= Getting Started with Project Syn
+
+This guide helps to get started with the various Project Syn tools.
+
+== Requirements
+
+Before you start, please make sure to have these requirements available:
+
+* A local Kubernetes cluster (k3s), managed with https://k3d.io/[k3d]
+* A https://gitlab.com[GitLab.com] account with an https://gitlab.com/profile/keys[SSH key configured]
+* `curl`
+
+== Kickstart Lieutenant
```

Comment:
Would it make sense to repeat the purpose for each component in the "getting started" guide as it might be the first "concrete" piece of documentation the user reads from from this project?

E.g: Lieutenant is the central inventory service all SYN-managed clusters report to, the "Mothership".

Reply:
Yes, that makes sense. I'll add this improvement to the next PR which will anyways add a description per component.